src/devices/src/virtio/net/device.rs

```diff
@@ -683,13 +683,11 @@ impl Net {
             // rate limiters present but with _very high_ allowed rate
             error!("Failed to get rx queue event: {:?}", e);
             METRICS.net.event_fails.inc();
+        } else if self.rx_rate_limiter.is_blocked() {
```

Comment:
Nit: I wouldn't have a separate commit for this. I would merge it in the 2nd or 3rd commit. Wherever it fits best.

Reply:
done!